Purchases.xcodeproj/project.pbxproj

```diff
@@ -1940,6 +1940,7 @@
 		2DC5622824EC63430031F69B /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				APPLICATION_EXTENSION_API_ONLY = YES;
```

Comment:
enabling this setting in the `swift_migration` PR catches this issue, so hopefully this will future-proof us.